modules/ROOT/pages/data-bridge.adoc

```diff
@@ -0,0 +1,141 @@
+----
++
+. The `cURL` command returns a response like this:
++
+----
+    {
+      "client_id": "5fafXXXXXXXXXXXXXXXXXXXXXXXX29c9",
+      "client_secret": "9509XXXXXXXXXXXXXXXXXXXXXXXXC10E",
+      "client_name": "ACM Integration App",
+      "redirect_uris": [
+        "https://[YourOrgDomain].my.salesforce.com/services/authcallback/Anypoint"
+      ],
+      "grant_types": [
+        "authorization_code",
+        "refresh_token"
+      ],
+      "public_keys": [],
+      "scopes": [
+        "offline_access",
+        "full"
+      ],
+      "enabled": true,
+      "owner_org_id": "f377XXXX-XXXX-XXXX-XXXX-XXXXXXXX9d08",
+      "as_id": "anypoint"
+    }
+----
++
+. Note the `client_id` and `client_secret` values.
+
+== Create an Authentication Provider in Your Salesforce Organization [[create-auth-provider]]
+
+NOTE: If your Anypoint Platform organization is hosted in the EU, change `+https://anypoint.mulesoft.com+` to `+https://eu1.anypoint.mulesoft.com+`.
+
+. In *Setup*, search for *Auth. Providers* using the *Quick Find* box and click *Auth. Providers* -> *New*.
+. In *Provider type* select *Open ID Connect*.
+. Set *Name* to `Anypoint`.
+. Set *Consumer Key* to the application ID of the connected app created in Anypoint Platform.
++
+This ID was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
++
+. Set *Consumer Secret* to the application secret of the connected app created in Anypoint Platform.
++
+This secret was returned by the Anypoint UI at the end of the task <<create-a-connected-app-with-the-user-interface>> or returned by the Anypoint API call at the end of the task <<create-a-connected-app-with-an-api-call>>.
++
+. Set *Authorize Endpoint URL* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/authorize+` .
+. Set *Token Endpoint URL* to `+https://anypoint.mulesoft.com/accounts/api/v2/oauth2/token+` .
+. Set *Default Scopes* to `full offline_access`.
+. Click *Save*.
+
+NOTE: Remember to associate the external data source with this authentication provider.
+
+== Authenticate the Data Bridge [[authenticate-data-bridge]]
+
+. Search for *External Data Source* in the *Quick Find* box and then click *External Data Sources*.
+. Locate the *Exchange* item and then click *Edit*.
+. Update *Identity Type* to *Named Principal*.
+. Update *Authentication Protocol* to *OAuth 2.0*.
+. Update *Authentication Provider* to *Anypoint*.
+. Click *Save*.
+. If you don't have an active session in your Anypoint Platform organization, the *Anypoint Login* page is shown. Log in to Anypoint Platform to complete the authentication. If you already have an active session, authentication is performed without requiring a login.
+. Verify the Exchange *External Data Sources* *Authentication Status* is *Authenticated*.
```

Comment:
```suggestion
. Verify that the authentication status of the Exchange *External Data Sources* is *Authenticated*.
```
"the Exchange *External Data Sources* what: section? pane? field? What kind of control does this label refer to?